src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -38,7 +38,7 @@ const resolveLandingPageLinks = (
       {
         ...HEALTH_TOOL_LINKS.MESSAGES[0],
         oldHref: mhvUrl(authdWithSSOe, 'secure-messaging'),
-        oldText: 'Inbox',
+        oldText: 'Go to inbox',
```

Comment:
You can omit this line, and the link will have the text from constants.js

Reply:
Actually, no. I had to change it here too, otherwise it will read just Inbox.

Reply:
I'm saying if you delete that line, the text _will_ read as "Go to inbox"

Reply:
Oh, that is very different. Thx

Reply:
Turns out I was completely wrong, here! Sorry!